scripts/OPNFV/NetworkServices/Methods.class/__methods__/deploy_cfns.rb

```diff
@@ -130,59 +126,55 @@ def create_template(name,nsd_properties,nsd_requirements)
           $evm.log(:error,"#{value} subnet not found in amazon")
           exit MIQ_ERROR
         end
-          nic<< value.gsub("\s", "").gsub("_", "")
-          security_group_name="DefaultSecurity"+value.gsub("\s", "").gsub("_", "")
+        nic<< value.gsub("\s", "").gsub("_", "")
+        security_group_name="DefaultSecurity"+value.gsub("\s", "").gsub("_", "")
         template_content["Resources"].merge!(security_group(security_group_name, network.ems_ref))
-          template_content["Resources"].merge!(template_interface(value.gsub("\s", "").gsub("_", ""), subnet.ems_ref, security_group_name))
-
+        template_content["Resources"].merge!(template_interface(value.gsub("\s", "").gsub("_", ""), subnet.ems_ref, security_group_name))
       end
-    end  
+    end
   end
   
-  
-  
   $evm.log(:info ,"nic array==> #{nic}")
-  template_content["Resources"].merge!(instance("Ec2Instance", nic, key_name, image.ems_ref,availability_zone,instance_type))  
+  template_content["Resources"].merge!(instance("Ec2Instance", nic, key_name, image.ems_ref,availability_zone,instance_type))
   $evm.vmdb('orchestration_template_cfn').create(
```

Comment:
this indentation was ok, when you do list or has and params on next line, the should be 2 spaces indentation, so call:

method(
  arg1,
  arg2)